endpoints/bahmni/input-validation.json

```diff
@@ -1,553 +1,203 @@
+          "uuid": {
+            "type": "string",
+            "format": "uuid"
           },
-          "uri": {
-            "type": "string"
-          }
-        }
-      },
-      "resourceVersion": {
-        "type": "string",
-        "required": true
-      }
-    },
-    "preferredAddress": {
-      "display": {
-        "type": "string"
-      },
-      "uuid": {
-        "type": "string"
-      },
-      "preferred": {
-        "type": "boolean"
-      },
-      "address1": {
-        "type": "string"
-      },
-      "address2": {
-        "type": "string"
-      },
-      "cityVillage": {
-        "type": "string"
-      },
-      "stateProvince": {
-        "type": "string"
-      },
-      "country": {
-        "type": "string"
-      },
-      "postalCode": {
-        "type": "string"
-      },
-      "countyDistrict": {
-        "type": "string"
-      },
-      "address3": {
-        "type": "string"
-      },
-      "address4": {
-        "type": "string"
-      },
-      "address5": {
-        "type": "string"
-      },
-      "address6": {
-        "type": "string"
-      },
-      "startDate": {
-        "type": "timestamp"
-      },
-      "endDate": {
-        "type": "timestamp"
-      },
-      "latitude": {
-        "type": "number"
-      },
-      "longitude": {
-        "type": "number"
-      },
-      "voided": {
-        "type": "boolean"
-      },
-      "address7": {
-        "type": "string"
-      },
-      "address8": {
-        "type": "string"
-      },
-      "address9": {
-        "type": "string"
-      },
-      "address10": {
-        "type": "string"
-      },
-      "address11": {
-        "type": "string"
-      },
-      "address12": {
-        "type": "string"
-      },
-      "address13": {
-        "type": "string"
-      },
-      "address14": {
-        "type": "string"
-      },
-      "address15": {
-        "type": "string"
-      },
-      "links": {
-        "type": "array",
-        "element": {
-          "rel": {
+          "identifier": {
             "type": "string"
           },
-          "uri": {
-            "type": "string"
-          }
-        }
-      },
-      "resourceVersion": "2.0"
-    },
-    "names": {
-      "type": "array",
-      "element": {
-        "display": {
-          "type": "string"
-        },
-        "uuid": {
-          "type": "string"
-        },
-        "givenName": {
-          "type": "string"
-        },
-        "middleName": {
-          "type": "string"
-        },
-        "familyName": {
-          "type": "string"
-        },
-        "familyName2": {
-          "type": "string"
-        },
-        "voided": {
-          "type": "boolean",
-          "default": false
-        },
-        "links": {
-          "type": "array",
-          "element": {
-            "rel": {
-              "type": "string"
-            },
-            "uri": {
-              "type": "string"
+          "resourceVersion": {
+            "type": "number"
```

Comment:
@bradsawadye Not a number

```suggestion
            "type": "string"
```

Reply:
fixed in https://github.com/jembi/openhim-mediator-mapping/pull/21/commits/da90c901b05f8fcd2480d722da8ccaa7f6a86d77